src/boards/include/boards/hellbender_0001.h

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// -----------------------------------------------------
+// NOTE: THIS HEADER IS ALSO INCLUDED BY ASSEMBLER SO
+//       SHOULD ONLY CONSIST OF PREPROCESSOR DIRECTIVES
+// -----------------------------------------------------
+
+#ifndef _BOARDS_HB_0001_H
+#define _BOARDS_HB_0001_H
```

Comment:
Could you change this from `_BOARDS_HB_0001_H` to `_BOARDS_HELLBENDER_0001_H` please? (so that it matches the actual filename)